tests/command_test.py

```diff
@@ -152,6 +152,90 @@ def test_call_test_float(self):
             }],
         )
 
+    def test_call_test_multiline_all(self):
+        self.snippet_call_test(
+            args=['-c', cat(), '-m', 'all'],
+            files=[
+                {
+                    'path': 'test/sample-1.in',
+                    'data': 'foo{0}foobar{0}'.format(os.linesep)
```

Comment:
```suggestion
                    'data': 'foo\nbar\n'.replace('\n', os.linesep)
```

`{0}` はなんだか見た目がこわいし、単に `replace` の方が素直でよい気がする